ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/PerformanceEnvelope.scala

```diff
@@ -21,55 +21,73 @@ import com.daml.ledger.api.v1.command_completion_service.{
   CompletionStreamResponse
 }
 import com.daml.ledger.api.v1.command_submission_service.SubmitRequest
-import com.daml.ledger.client.binding.{Primitive => P}
 import com.daml.ledger.api.v1.commands.{Command, Commands}
 import com.daml.ledger.api.v1.ledger_offset.LedgerOffset
 import com.daml.ledger.api.v1.transaction.Transaction
 import com.daml.ledger.api.v1.transaction_filter.{Filters, TransactionFilter}
 import com.daml.ledger.api.v1.transaction_service.{GetTransactionsRequest, GetTransactionsResponse}
-import io.grpc.{Context, Status}
+import com.daml.ledger.client.binding.{Primitive => P}
+import com.daml.ledger.test.performance.{PingPong => PingPongModule}
 import io.grpc.stub.StreamObserver
+import io.grpc.{Context, Status}
+import org.slf4j.Logger
 import scalaz.syntax.tag._
 
 import scala.collection.concurrent.TrieMap
 import scala.concurrent.{ExecutionContext, Future, Promise, blocking}
 import scala.util.{Failure, Random, Success, Try}
-import com.daml.ledger.test.performance.{PingPong => PingPongModule}
-import org.slf4j.Logger
 
 sealed trait Envelope {
   val name: String
   val transactionSizeKb: Int
   val throughput: Int
   val latencyMs: Int
+  val inFlightLimit: Int
```

Comment:
@dasormeter : AFAIS, in a successful test for a specific envelope it  always holds that `numInflightRequests <= throughput * latencyMs / 1000`. So, I'd suggest to not introduce the inFlightLimit property, but instead add a function `maxRequestsInflight` that computes it as per the above formula.

Reply:
i.e.  `maxRequestsInFlight = throughput * latencyMs / 1000`